tests/ui/test_devhub.py

```diff
@@ -33,3 +35,26 @@ def test_devhub_addon_edit_link_works(base_url, selenium, devhub_login):
 def test_devhub_addon_upload(base_url, selenium, devhub_upload):
     """Test uploading an addon via devhub."""
     'ui-test-addon-2' in devhub_upload.addons[-1].name
+
+
+@pytest.mark.fxa_login
+@pytest.mark.desktop_only
+@pytest.mark.nondestructive
+@pytest.mark.withoutresponses
+def test_devhub_logout(base_url, selenium, devhub_login):
+    """Logging out from devhub."""
+    assert devhub_login.logged_in
+    devhub = devhub_login.header.click_sign_out()
+    assert not devhub.logged_in
+
+
+@pytest.mark.desktop_only
+@pytest.mark.nondestructive
+@pytest.mark.withoutresponses
+def test_devhub_register(base_url, selenium):
+    """Test register link loads register page."""
+    selenium.get('http://olympia.test/en-US/developers/')
```

Comment:
Why are you requesting this resource? This doesn't return anything or doesn't have any side effects.

Reply:
Probably a copy/paste error.